addons/html_editor/static/tests/insert/html.test.js

```diff
@@ -150,6 +150,17 @@ describe("collapsed selection", () => {
         });
     });
 
+    test('should paste an "empty" block', async () => {
+        await testEditor({
+            contentBefore: "<p>abcd[]</p>",
+            stepFunction: async (editor) => {
+                editor.shared.domInsert(parseHTML(editor.document, "<p>efgh</p><p></p>"));
+                editor.dispatch("ADD_STEP");
+            },
+            contentAfter: "<p>abcdefgh</p><p>[]<br></p>",
```

Comment:
I'm not sure I get it. The commit title speaks about pasting empty nodes but the test don't use "paste". This is an `insert` fix, right? So did you mean "insert" in the commit title rather than "paste"?

Reply:
Agreed, it should have been `insert` instead of `paste`. I'll update it.